apache/superset
# feat: Migrates Dual Line Chart to Mixed Chart

### SUMMARY
This PR migrates the Dual Line Chart to Mixed Chart (ECharts).

- Removes the legacy chart code and unregisters the plugin.
- Removes old plugin references throughout the codebase
- Updates unit tests
- Updates E2E tests
- Updates the example dashboards
- Updates translation files

### AFTER SCREENSHOTS
<img width="1084" alt="Screenshot 2023-05-03 at 18 07 54" src="https://user-images.githubusercontent.com/70410625/236050273-d707ac1d-718a-4d87-a7d7-bb8894c9ef7a.png">

### TESTING INSTRUCTIONS
1 - Make sure all Dual Line charts were converted to Mixed charts
2 - Make sure Dual Line is not available anymore in the viz picker
3 - Make sure that it's possible to revert the migration by executing superset db downgrade and reverting this PR

### ADDITIONAL INFORMATION
1 - Make sure all Treemap (legacy) charts were converted to Treemap
2 - Make sure Treemap (legacy) is not available anymore in the viz picker
3 - Make sure that it's possible to revert the migration by executing superset db downgrade and reverting this PR
